sdk/src/azure/iot/az_iot_hub_client_twin.c

```diff
@@ -173,8 +173,17 @@ AZ_NODISCARD az_result az_iot_hub_client_twin_parse_received_topic(
       {
         // Is a reported prop response
         out_response->response_type = AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_REPORTED_PROPERTIES;
-        _az_RETURN_IF_FAILED(az_iot_message_properties_find(
-            &props, az_iot_hub_twin_version_prop, &out_response->version));
+
+        result = az_iot_message_properties_find(
+            &props, az_iot_hub_twin_version_prop, &out_response->version);
+        if (result == AZ_ERROR_ITEM_NOT_FOUND)
+        {
+          out_response->version = AZ_SPAN_EMPTY;
+        }
+        else
+        {
+          _az_RETURN_IF_FAILED(result);
+        }
```

Comment:
Are we sure this is what we want? We are changing `AZ_ERROR_ITEM_NOT_FOUND` return value to `AZ_OK` on line 195 now.

Do we need to update the docs here:
https://github.com/Azure/azure-sdk-for-c/blob/cc7beafdb56f2f495d488451ca2f28f855070a17/sdk/inc/azure/iot/az_iot_hub_client.h#L625-L632

Reply:
We were notified via a colleague on Azure RTOS who had a customer issue.  Through debugging, it was found that IoT Edge did not supply the $version in the reported properties response while IoT Hub did.  This was confirmed by IoT Edge.

Can you please remind me where I create the changelog entry? Thanks!

Reply:
> Can you please remind me where I create the changelog entry? Thanks!

Here:
https://github.com/Azure/azure-sdk-for-c/blob/main/CHANGELOG.md#bugs-fixed